SYCL/Basic/query.hpp

```diff
@@ -43,5 +43,5 @@ int main() {
       }
     }
   }
-  return 0;
+  return 1;
```

Comment:
is this right?  I believe the convention is that main() should return 0 if there is no error. 

I don't see why we should return 1 here, but if I'm overlooking a valid reason, we should probably add a comment.

Reply:
Hi, no, it is in progress now, variables to duplicate devices don't work on ci